README.md

```diff
@@ -85,6 +85,7 @@ To run this demo locally, add an `.env` file with the following variables:
 ```
 REACT_APP_DAILY_BASE_URL=<-your Daily URL, e.g. https://demo-example.daily.co/->
 REACT_APP_DAILY_API_KEY=<-your Daily API key->
+REACT_APP_ROOM_ENDPOINT=local
```

Comment:
Is `local` meant to be a string here? The comment on line 12 of `.env.sample` might be miscuing, if not.

Reply:
I've changed line 12 of `.env.sample`